terraform/aws/ec2.tf

```diff
@@ -2,7 +2,6 @@ resource "aws_instance" "web_host" {
   # ec2 have plain text secrets in user data
   ami           = "${var.ami}"
   instance_type = "t2.nano"
-
   vpc_security_group_ids = [
```

Comment:
```suggestion
  monitoring = true
}
```
<details>
    <summary>
        <img src="https://bc-network-graph-icons.s3.us-west-2.amazonaws.com/prCommentsSeverityIcons/medium.svg" alt="MEDIUM" width="64px" align="center">
            &nbsp;&nbsp;<b style="color:black" >Detailed monitoring for EC2 instances is disabled</b>
            <br>
               &nbsp;&nbsp;&nbsp;&nbsp;Resource: <a href="https://github.com/bds1959/terragoat/blob/bds-devops/terraform/simple_instance/ec2.tf#L19-L26">aws_instance.web_server_instance</a>  |  ID: <code>BC_AWS_LOGGING_26</code>
            <br></summary>
<h4>Description</h4>
TBA

</details>